src/lib/mdns/Discovery_ImplPlatform.cpp

```diff
@@ -49,24 +49,14 @@ CHIP_ERROR DiscoveryImplPlatform::Init()
     {
         ReturnErrorOnFailure(ChipMdnsInit(HandleMdnsInit, HandleMdnsError, this));
         mCommissionInstanceName = GetRandU64();
-        mMdnsInitialized        = true;
     }
 
     return CHIP_NO_ERROR;
 }
 
 CHIP_ERROR DiscoveryImplPlatform::Start(Inet::InetLayer * inetLayer, uint16_t port)
 {
-    ReturnErrorOnFailure(Init());
-
-    CHIP_ERROR error = ChipMdnsStopPublish();
-
-    if (error != CHIP_NO_ERROR)
-    {
-        ChipLogError(Discovery, "Failed to initialize platform mdns: %s", ErrorStr(error));
-    }
-
-    return error;
+    return Init();
```

Comment:
If the arguments are ignored,  then why do they exist?

Reply:
It was already refactored in the PR: https://github.com/project-chip/connectedhomeip/pull/10181, so I will just rebase on it after merging.